From 45ceaf931176949f6d1fe7fbd8ff870e5bdb72a2 Mon Sep 17 00:00:00 2001
From: Leon Anavi <leon.anavi@konsulko.com>
Date: Mon, 19 Aug 2024 13:41:38 +0000
Subject: [PATCH] orangepi_zero_defconfig: RAUC

Configure U-Boot environment location, size (CONFIG_ENV_SIZE) and
offset (CONFIG_ENV_OFFSET) on the FAT partition for Orange Pi Zero
and after that use it in /etc/fw_env.config.

Upstream-Status: Inappropriate [RAUC specific]

Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 configs/orangepi_zero_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configs/orangepi_zero_defconfig b/configs/orangepi_zero_defconfig
index abdf9a9396..3eb721cd1b 100644
--- a/configs/orangepi_zero_defconfig
+++ b/configs/orangepi_zero_defconfig
@@ -15,3 +15,6 @@ CONFIG_SUN8I_EMAC=y
 CONFIG_SPI=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
+CONFIG_ENV_SIZE=0x20000
+CONFIG_ENV_OFFSET=0x0000
+CONFIG_ENV_IS_IN_FAT=y
-- 
2.45.2

